configure

```diff
@@ -787,6 +787,9 @@ def configure_arm(o):
 
   o['variables']['arm_fpu'] = options.arm_fpu or arm_fpu
 
+  if flavor == 'win':
+    o['msvs_windows_sdk_version'] = 'v10.0'
```

Comment:
There's just a wierd hack at https://github.com/nodejs/node-chakracore/blob/xplat/configure#L1355 for the `['variables']` that always confuses me